### PR TITLE
Issues pull from Anidata now, index.html edited

### PR DESCRIPTION
**Pull request by coxjonc**
Issues and labels render just fine now. Currently working on getting the language filters at the top of the page to work and should have that done shortly. Also looking into adding filters by issue difficulty.

### CODE REVIEW

File: index.html
Comment (dlrobertson) [inline index.html]: This can just say "Anidata Starters"

File: index.html
Comment (dlrobertson) [inline index.html]: Same

File: index.html
Comment (dlrobertson) [inline index.html]: href should probably be the anidata project page `https://github.com/anidata/`. right? Or maybe this should be a list of our ongoing projects. I'd lean towards making this easier and making this the group github page though.

File: site.js
Comment (dlrobertson) [inline site.js]: Wouldn't this be `lang:python`?

File: site.js
Comment (dlrobertson) [inline site.js]: This should be updated to anidata, but I'm not sure which repo should be used as the "reference" repo yet.

File: site.js
Comment (dlrobertson) [inline site.js]: Same.

**Review by dlrobertson — COMMENTED** [review]
Thanks! This is awesome work! I have a few nits and comments.

There are a few infrastructure things we have to figure out that might be blocking this PR a bit. Mainly figuring out what our "reference" repo is.

@bmenn any thoughts on this?

**Comment by coxjonc** [comment]
See 2f07e - I think that should address most of the issues.
One strange problem I've encountered is that although the issues url is set to `https://api.github.com/search/issues` the JSON response seems to include pull requests as well as issues. Is that the Github API's normal behavior or am I doing something wrong?

**Comment by dlrobertson** [comment]
> https://api.github.com/search/issues the JSON response seems to include pull requests as well as issues. Is that the Github API's normal behavior or am I doing something wrong?

No that's just how the API works. PRs and Issues never have overlapping numbers. If you were to go to [https://github.com/anidata/servo-starter/issues/1](https://github.com/anidata/servo-starters/issues/1), it would take you here.

**Comment by bmenn** [comment]
I don't know if a list of labels might help. I don't have one for all the repos I've been working with but this bash one-line could help (I used it to help sync tags across repos):

```
curl -u bmenn:$(cat github-token) https://api.github.com/repos/anidata/ht-archive/labels | jq -r '.[] | {"name": .name, "color": .color} | tostring' > github-labels.json
```

**Comment by bmenn** [comment]
@danlrobertson @geezhawk 

If we need something in a pinch, maybe anidata/ht-archive could be a reference repo? Otherwise, I think making anidata/anidata.github.io the reference repo or look into adding the ability to have multiple reference repos. Thoughts?

**Comment by coxjonc** [comment]
@bmenn I think making ht-archive the reference repo should work fine. What do you think @danlrobertson? Also, do you guys think I should filter out pull requests? If the point of this tool is to help beginners find issues to contribute to it doesn't really make sense to require them to scroll through a bunch of pull requests to find an open issue.

**Comment by dlrobertson** [comment]
I think `ht-archive` would be a reasonable choice. After thinking about it, IMO at some point the labels should be moved to the anidata.github.io and that should be the reference, but that could be an update.

> Also, do you guys think I should filter out pull requests?

Definitely

**Comment by coxjonc** [comment]
I think c4beb6c fixes all your issues but let me know if there are other concerns you want to address before merging.

**Review by dlrobertson — APPROVED** [review]
:+1:

**Pull request merged**
